README.md

```diff
@@ -45,8 +48,9 @@ import (
 var VERSION string
 
 func main() {
-      // Create a new CLI app with urfave/cli
-      app := cli.NewApp()
+      // Create a new CLI app. This will return a urfave/cli App with some
+      // common initialization.
+      app := entrypoint.NewApp()
```

Comment:
Make sure to call out this change in the release notes!

Reply:
Just to clarify, you mean https://github.com/gruntwork-io/gruntwork-cli/releases and newsletter and not a special file in here that tracks changes? Also, this should be a minor release (`0.2.0`) because it is a backwards compatible feature release right?

Reply:
Yes on all counts!